.github/pull_request_template.md

```diff
@@ -22,8 +22,12 @@
   SCOPE should be one of:
     - components
     - components-native
+    - deps
+    - deps-dev
     - design
+    - docx
     - eslint
+    - formatters
```

Comment:
My thinking was that adding these options in the docs will help us keep inventory of all available scopes for future references and resolutions. As you can see, I didn't add these options to the `PullRequestGenerator.tsx`, since I agree that there's probably no need to display them in the UI tool.